pkg/api/api.go

```diff
@@ -165,6 +165,10 @@ func (hs *HTTPServer) registerRoutes() {
 		)
 	}
 
+	if hs.Cfg.PluginAdminEnabled && (hs.Features.IsEnabledGlobally(featuremgmt.FlagDashboardRestore)) {
+		r.Get("/dashboard/recently-deleted", reqSignedIn, hs.Index)
+	}
```

Comment:
What does `hs.Cfg.PluginAdminEnabled` mean, and why is recently-deleted gated behind it?

Reply:
From what I found out I was assuming that the route would only be accessible for admins but I misread this. So I removed it.